File: src/engsoft/jogo/patos/Pular_Baixo.java
package engsoft.jogo.patos;

public class Pular_Baixo implements Padrao_Pular{
	
	private double altura;
	
	public Pular_Baixo()
	{
		altura = 1;	
	}
	
    @Override
	public String pular() {
		return "Pulando como uma pulga. Altura: " + getAltura();		
	}

    @Override
	public double getAltura() {
		return altura;
	}	
}

File: Pular_Baixo.ts
import { Padrao_Pular } from "./Padrao_Pular";

export class Pular_Baixo implements Padrao_Pular {
    pular(): string {
        return "Pulo baixo!";
    }

    getAltura(): number {
        return 1;
    }
}
